### ISSUE TITLE
[bug] no security scan

### ISSUE THREAD

**Issue by mcandre** [conan-io/conan#10938]
### Steps to reproduce (Include if Applicable)

```console
$ conan audit
'audit' is not a Conan command. See 'conan --help'.

ERROR: Unknown command 'audit'
```

The Xray integration is unhelpful: It requires a tremendous amount of extra work for the user to setup, before they can begin to generate basic CVE reports of their Conan dependency trees. Which means essentially no one is scanning their C/C++ projects for vulnerabilities. We need a simpler way, akin to `npm audit`, `gem audit`, `safety check`, `gradle audit`.

**Comment by memsharded** [comment]
Conan does not implement security scanning, and it is not in the roadmap for the client. Conan is open source and the server side is implemented by some different vendors. Yes, XRay provide some information for vulnerabilities, but the Conan API cannot be specialized to work only with Artifactory/XRay, and not with other servers. Maybe Conan 2.0 with its new custom command layers will allow for some specialized commands that work with different platforms, and something can be done there specific for Artifactory/XRay. But that will belong to other place, not the core Conan client.

I suggest if you have suggestions or feedback how to improve the XRay support, to send them to the XRay team.

Closing this as this is not a bug and not a feature that can be implemented here at this moment. Thanks for the feedback.

**Comment by mcandre** [comment]
Enjoy your malware.

**Comment by czoido** [comment]
Hi!, 

We have officially announced the new `conan audit` command, available starting from Conan 2.14. This command allows you to scan your dependencies for known CVEs.  

For more details, please refer to our [blog post](https://blog.conan.io/introducing-conan-audit-command/).  

In Short...  

1. **Register for free** at [audit.conan.io/register](https://audit.conan.io/register).  
2. **Save your token** and **activate it** via the confirmation email you receive.  
3. **Configure Conan to use your token**:  

  ```bash
  conan audit provider auth conancenter --token=<token>
  ```
  
4. **Scan** for vulnerabilities:

  ```bash
  # Check a specific reference
  conan audit list zlib/1.2.13
  
  # Scan the entire dependency graph
  conan audit scan .  # Path to the conanfile.py/txt
  ```
